### PR TITLE
feat: i used base, menu and noscript elements fix #80 fix#81 fix #82

### LINKED ISSUE #80: the `<menu>` tag

### LINKED ISSUE #82: the `<noscript>` tag

### PR DESCRIPTION
**Pull request by antonia2203**

### CODE REVIEW

File: src/antonia2203/menu.pug [hunk 1]
```diff
@@ -0,0 +1,22 @@
+html 
+  head 
+    title The menu, base and noscript elements
+    base(href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu", target="_blank" )
```
Comment (MariaTeo) [inline src/antonia2203/menu.pug]: make it look pretty

Comment (MariaTeo) [reply src/antonia2203/menu.pug]: please 🥺

**Review by MariaTeo — CHANGES_REQUESTED** [review]
🥺

File: src/antonia2203/menu.pug [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  body 
+    h1 The base element 
+    p 
+      a(
+        href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base"
+      ) Da click pe mine
+    h2 A menu
+      menu
```
Comment (MariaTeo) [inline src/antonia2203/menu.pug]: it shouldnt be a child of h2

**Review by Alez05 — APPROVED** [review]
🥺🥺

**Review by MariaTeo — APPROVED** [review]
yayy 😼🤑

**Review by flivanov92 — COMMENTED** [review]
Nice

**Review by flivanov92 — COMMENTED** [review]
👍

**Pull request merged**